packages/fe-container/src/index.tsx

```diff
@@ -28,7 +28,13 @@ function retrieveContainer (props: any) {
 }
 
 function render (props: any) {
-  ReactDOM.render(<Composer configurationName={props.configurationName}/>, retrieveContainer(props))
+  ReactDOM.render(
+    <Composer
+      configurationName={props.configurationName}
```

Comment:
Is it possible to use the spread operator on props?

Reply:
it's kinda useless to spread if I have to take the ...rest and pass it again to line 36. best i can do it's to take also props.container which is used at line 25. looks messy to me